docs/docsite/rst/roadmap/ROADMAP_2_4.rst

```diff
@@ -1,224 +1,220 @@
-
-  - Create import_x for ‘static includes’ (import_task, import_play, import_role)
-
-    - Any directives are applied to the ‘imported’ tasks
-
-  - Create include_x for ‘dynamic includes’ (include_task, include_role)
-
-    - Any directives apply to the ‘include’  itself
-
-- **Windows Support**
-
-  - New PS/.NET module API
-  - Windows Nano Server support
-  - Windows module_utils pluginloader
-  - Refactor duplicated module code into new module_utils files
-  - Evaluate #Requires directives (existing and new: PS version, OS version, etc)
-  - Improve module debug support/persistence
-  - Explore official DSC support
-  - Explore module intermediate output
-  - Explore Powershell module unit testing
-  - Explore JEA support (stretch)
-  - Extended become support with network/service/batch logon types
-  - Module updates
-
-    - Split "Windows" category into multiple subs
-    - Domain user/group management modules
-    - win_mapped_drive module
-    - win_hotfix
-    - win_updates rewrite to require become
-    - win_package changes required to deprecate win_msi
-    - win_copy re-write
-
-- **Cloud Provider Support**
-
-  - AWS
-
-    - Focus on pull requests for various modules
-    - Triage existing merges for modules
-    - Module work
-
-      - elb-target-groups
-      - alb*
-      - ecs
-      - Data Pipelines
-      - VPN
-      - DirectConnect
-
-  - Azure
-
-    - Expose endpoint overrides
-    - Reformat/document module output to collapse internal API structures and surface important data (eg, public IPs, NICs, data disks)
-    - Add load balancer module
-    - Add Azure Functions module
-
-  - Google Cloud Platform
-
-    - New Module: DataProc
-    - Support for Cross-Region HTTP Load Balancing
-    - New Module: GKE
-
-- **Network Roadmap**
-
-  - Removal of ``*_template`` modules
-  - Session Tracing
-  - Refactor ansible-connection to cli
-  - Module Work
-
-    - Declarative intent modules
-    - OpenVSwitch
-
-- **Contributor Quality of Life**
-
-  - All Core and Curated modules will work towards having unit testing.
-  - More bot improvements!
-  - Test Infrastructure changes
-
-    - Shippable + Bot Integration
-
-      - Provide verified test results to the bot from Shippable so the bot can comment on PRs with CI failures.
-      - Enable the bot to mark PRs with ``ci_verified`` if all CI failures are verified.
-
-    - Windows Server 2016 Integration Tests
-
-      - Restore Windows Server 2016 integration tests on Shippable.
-
-        - Originally enabled during the 2.3 release cycle, but later disabled due to intermittent WinRM issues.
-        - Depends on resolution of WinRM connection issues.
+============================
+**Target: Aug/Mid-September 2017**
+
+Administrivia and Process
+-------------------------
+- Starting with 2.4, all items that are deprecated will be removed in 4 major releases unless otherwise stated.
+  - For example: A module that is deprecated in 2.4 will be removed in 2.8
```

Comment:
Blank line needed here.
In RST blank lines are needed when changing the indent level on bullet points.